pkg/support/dump_windows_test.go

```diff
@@ -41,15 +41,15 @@ func TestDumpLog(t *testing.T) {
 	fs.MkdirAll(logDir, os.ModePerm)
 	fs.MkdirAll(antreaWindowsOVSLogDir, os.ModePerm)
 	fs.MkdirAll(antreaWindowsKubeletLogDir, os.ModePerm)
-	fs.Create(filepath.Join(logDir, "rancher-wins-antrea-agent.log"))
+	fs.Create(filepath.Join(logDir, "antrea-agent.exe.log"))
```

Comment:
the tests have passed so we can merge this, but is there a specific reason for adding the `.exe` to the file name?

Reply:
No specific reasons, just because the agent bits is named as "antrea-agent.exe" on Windows. So I use its Windows style name prefix here.

Reply:
I was asking because it doesn't match what we do for kubelet for example (I see `kubelet.log` below)

Reply:
Do you mean you want their names consistent?

Reply:
I realize now that this is just the unit test, and that in practice we are copying the files as they are on the host's filesystem.

I guess my only preference would be to rename `antrea-agent.exe.log` to `antrea-agent.exe.INFO` in the unit test, so that it better matches the actual log file name, based on what you mentioned above.

If this is the only change, we don't need to run e2e tests again before merging, we just need to make sure that unit tests are still passing.